### ISSUE TITLE
Insecure connection could allow RCE

### ISSUE THREAD

**Issue by ShaKabosh** [bajrangCoder/acode-plugin-acodex#9]
If the server is installed on Termux, then it seems as though anyone could, knowing your IP address, easily connect to the websocket unauthenticated and have RCE through Termux, which is *especially* bad if your phone is rooted.

**Comment by bajrangCoder** [comment]
Yes, but this is local server no one can access other than you, but if you will give your phone then  it may be insecure.

Or if you have any suggestions for this , you can provide. i will happy to see.

**Comment by 14725** [comment]
If we use the server and also surfing the web...

Then a evil (?) page surely knows your one of your IP as 127.0.0.1, and he cound connect to it easily by start a WebSocket and grab a shell of your phone... 

Port could be scanned, too.

**Comment by bajrangCoder** [comment]
Ok , In next update I will add authentication system ♥️♥️
